spec/entry_spec.rb

```diff
@@ -54,10 +54,28 @@
     it 'can handle multiple locales' do
       vcr('entry_locales') {
         cat = create_client.entries(locale: "*").items.first
-        expect(cat.fields[:name][:'en-US']).to eq "Nyan Cat"
-        expect(cat.fields[:name][:'es']).to eq "Gato Nyan"
+        expect(cat.fields('en-US')[:name]).to eq "Nyan Cat"
+        expect(cat.fields('es')[:name]).to eq "Gato Nyan"
```

Comment:
You're right, this is actually the documented way, at least for `sync`: <https://github.com/contentful/contentful.rb/blame/master/README.md#L327>

I still think we should make both methods consistent in their use of symbols vs. strings for the locales, otherwise it'll be confusing.